src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -166,7 +166,7 @@ public static class ScriptConstants
         public const string LinuxMetricEventStreamName = "MS_FUNCTION_METRICS";
         public const string LinuxFunctionDetailsEventStreamName = "MS_FUNCTION_DETAILS";
         public const string LinuxAzureMonitorEventStreamName = "MS_FUNCTION_AZURE_MONITOR_EVENT";
-        public const string LinuxMSISpecializationStem = "/api/specialize?api-version=2017-09-01";
```

Comment:
I think you might have mentioned this elsewhere. but just to confirm :)
1)we will only release this change after new token svc image is already released fully
2)the new token svc image supports the old api version 2017-09-01.

Correct?

Reply:
Yes for both. Finished new token service release yesterday 3/16 across all of public.